core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,29 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    abort-all-workflows-activity {
+      depends-on = [service-unbind]
+      timeout = 1 hour
```

Comment:
The previous timeout was `Infinity` which still makes this one infinitely shorter but yeah I don't know what a sensible value would be here :/